README.md

```diff
@@ -42,6 +42,18 @@ All typescript rules compatible with the AWS CDK structure have been added separ
 }
 ```
 
+### Setup for React tests
+
+Rules specific to tests written in React are added separately and can be integrated by using the following configurations. 
+
+```json
+{
+    "extends": ["eslint-config-lucideus", "eslint-config-lucideus/react-tests"]
+}
+```
+
+*Note*: For the above rules to apply, make sure you have enabled linting in for test files/folder.
```

Comment:
@Bhavin789 Is there a specific configuration needed to satisfy this? If yes, possible to include an example here itself